app/actions/ui_actions.js

```diff
@@ -23,13 +23,42 @@ export const {
     pageLoaded,
     focusWebview
 } = createActions(
-    TYPES.SHOW_SETTINGS_MENU,
-    TYPES.HIDE_SETTINGS_MENU,
-    TYPES.BLUR_ADDRESS_BAR,
-    TYPES.DESELECT_ADDRESS_BAR,
-    TYPES.SELECT_ADDRESS_BAR,
-    TYPES.RESET_STORE,
-    TYPES.RELOAD_PAGE,
-    TYPES.PAGE_LOADED,
-    TYPES.FOCUS_WEBVIEW
+    {
+        SHOW_SETTINGS_MENU : [
+            undefined,
```

Comment:
@manavbp I'm not sure if the payload creator can be `undefined` here... Can you check w/r/t https://redux-actions.js.org/api/createaction#createactionsactionmap

Reply:
createAction(type, payloadCreator, MetaCreator)
payloadCreator must be a function, undefined, or null. If payloadCreator is undefined or null, the identity function is used.
https://redux-actions.js.org/api/createaction#createactiontype-payloadcreator